docs/guidance/build.md

```diff
@@ -48,7 +48,18 @@ Your choice of architecture can determine how frequently your data is updated, w
 ```
 **Resource:** [Technical case studies: OCDS implementation insights report](https://www.open-contracting.org/resources/technical-case-studies-ocds-implementation-insights/) provides insights into the technical choices made in OCDS implementations in Paraguay, Zambia, Colombia, Moldova and Argentina's Road Agency Vialidad.
 
-If you’re reusing an existing tool for collecting OCDS data, you should consider how you will collate and publish data collected using the tool. The [OCDS Helpdesk](../../support/#ocds-helpdesk) can help you to consider options for publication.
+### Decide how to combine spreadsheet data
+
+If you aren't creating or updating an IT system, but are instead collecting data using multiple spreadsheets, then this step is about working out how to combine your data into a single file for publication. Combining your data makes it easier for users to analyze the whole dataset.
+
+If you plan to publish your data infrequently and you only have a small number of spreadsheets, then simply copy-pasting the data into a single file for publication may be the easiest method.
```

Comment:
This assumes the headers are identical. Readers unfamiliar with tabular data either won't understand what it means to copy-paste the data into a single file, or will try to copy-paste data from files with different headers.

Reply:
I'm also unclear how the process leads to there being multiple spreadshets.

Reply:
> I'm also unclear how the process leads to there being multiple spreadshets.

There would be multiple spreadsheets if different users/departments/ agencies fill-in offline spreadsheets and send them to a central point for reporting purposes.

We have seen this being done in at least Nigeria (before NOCOPO, each procuring entity filled out a spreadsheet and sent it to the Bureau of Public Procurement) and the Philippines (each procuring entity fills out a spreadsheet and sends it to the Government Procurement Policy Board) so it seems like a common pattern where there isn't a central IT system.

The MS Excel format version of the OCDS data collection spreadsheet is designed with this use case in mind.

How about:

```
If you aren't creating or updating an IT system, but are instead collecting data from different individuals, departments or agencies using spreadsheets, then this step is about working out how to combine your data into a single file for publication. Combining your data makes it easier for users to analyze the whole dataset.
```

> This assumes the headers are identical. Readers unfamiliar with tabular data either won't understand what it means to copy-paste the data into a single file, or will try to copy-paste data from files with different headers.

If the publisher is using a data collection spreadsheet I would expect the headers to be the same across all copies of the template, but happy to clarify if you think it's important. How about:

```
If you plan to publish your data infrequently, you only have a small number of spreadsheets and your spreadsheets have identical headers, then simply copy-pasting the data into a single file for publication may be the easiest method.
```

Reply:
> There would be multiple spreadsheets if different users/departments/ agencies fill-in offline spreadsheets and send them to a central point for reporting purposes.

> We have seen this being done in at least Nigeria (before NOCOPO, each procuring entity filled out a spreadsheet and sent it to the Bureau of Public Procurement) and the Philippines (each procuring entity fills out a spreadsheet and sends it to the Government Procurement Policy Board) so it seems like a common pattern where there isn't a central IT system.

It would be good to have 1-2 sentences to describe this pattern somewhere in these docs, so that readers are clear on what spreadsheets are being combined.

Reply:
> If you aren't creating or updating an IT system, but are instead collecting data from different individuals, departments or agencies using spreadsheets, then this step is about working out how to combine your data into a single file for publication. Combining your data makes it easier for users to analyze the whole dataset.

Yes, that satisfies my above comment.

Reply:
> If you plan to publish your data infrequently, you only have a small number of spreadsheets and your spreadsheets have identical headers, then simply copy-pasting the data into a single file for publication may be the easiest method.

+1. In my experience, users frequently modify, add and remove headings without recognizing the impact.